e2e-tests/cypress/e2e/journeys/insurance/eligibility/need-to-start-again.spec.js

```diff
@@ -82,10 +83,10 @@ context('Insurance Eligibility - Need to start again exit page', () => {
   });
 
   describe('clicking the submit button', () => {
-    it(`should redirect to ${ROUTES.INSURANCE.START}`, () => {
+    it(`should redirect to ${ROUTES.INSURANCE.buyerCountryRoute}`, () => {
       submitButton().click();
 
-      cy.url().should('include', ROUTES.INSURANCE.START);
+      cy.url().should('include', buyerCountryRoute);
```

Comment:
This is my fault for not doing this, but can we use `eq` instead of `include`? e.g

```js
const expected = `${Cypress.config('baseUrl')}/the-url;

cy.url().should('eq', expected);
```

Reply:
done